eng/scripts/Automation-Update-Metadata.ps1

```diff
@@ -0,0 +1,41 @@
+# Automation-Update-Metadata.ps1
+# This script is used to update metadata for the Azure SDK automation process
+
+param(
+    [Parameter(Mandatory = $true)]
+    [string]$sdkRepoPath,  # Absolute path to the root folder of the local SDK repository.
+    
+    [Parameter(Mandatory = $true)]
+    [string]$packagePath   # Absolute path to the root folder of the local SDK project.
+)
+
+# Step 1: Find the csproj file in the src directory
+$srcPath = Join-Path $packagePath "src"
+$csprojFile = Get-ChildItem -Path $srcPath -Filter "*.csproj" | Select-Object -First 1
```

Comment:
You don't need to full csproj file path. You just need the path to the project so if `$packagePath/src` assuming src isn't already in the package path.